api-spec/extensions/query/query.fragment.yaml

```diff
@@ -79,5 +79,3 @@ components:
               items:
                 type: string
               description: Find items with a property that matches one of the specified strings. A case-insensitive comparison must be performed.
```

Comment:
`time` got inadvertently re-added when the branch with the `in` operator was merged with the commit that removed `time`